### DISCUSSION TITLE
Now that PlanetScale is GA, does it make sense to continue this discussion group in the beta repo?

### DISCUSSION THREAD

**Discussion by jonrcrowell** [planetscale/discussion#142] [Q&A] [answered]
Are there any plans to start a Discord channel?

**Comment by tbarn** [comment]
Hey @jonrcrowell! Stay tuned. We are going to be working on some new things for the PlanetScale community in 2022. 

What are things you would like to get out of a discussion group, chat, etc for PlanetScale? Anything in particular you'd like to see from us?

**Reply by patrick91** [reply]
hey, did you end up creating a discord in the end? I'd love to chat with the team and other people using planetscale 😊

**Reply by tbarn** [reply]
Hey! @CanRau @patrick91 Today Twitter is the place to find the most PlanetScale users and team members talking about PlanetScale. We will still keep Discord in mind for the future.

**Reply by CanRau** [reply]
Alright, is it appreciated to ping @planetscale on Twitter about an open discussion I'm very curious about? 😇

**Reply by tbarn** [reply]
Yeah, we don't mind! @planetscaledata will be the handle you are looking for.

**Reply by ImLunaHey** [reply]
Any update on this? Happy to make a unofficial server if need be.

**Comment by CanRau** [comment]
Was also looking for a Discord as it's pretty helpful, I'm in so many dev Discords, and they feel way more alive (usually) then GH Discussions, I think 🤔

**Reply by imjakechapman** [reply]
Just googled 'planetscale discord' 5 seconds ago and landed here. Sad day.

**Comment by jgreet** [comment, marked as answer]
We opened up a PlanetScale Discord server this week: https://discord.gg/entrSsZHTV 🚀

**Reply by wscotten** [reply]
Invite expired

**Reply by ImLunaHey** [reply]
unlimited invite link: https://discord.gg/mjz3NSc2rv

**Reply by jgreet** [reply]
thanks, updated with one that won't expire
